packages/driver/src/cypress/log.ts

```diff
@@ -18,94 +18,96 @@ const BLACKLIST_PROPS = 'snapshots'.split(' ')
 
 let counter = 0
 
-// mutate attrs by nulling out
-// object properties
-const reduceMemory = (attrs) => {
-  return _.each(attrs, (value, key) => {
-    if (_.isObject(value)) {
-      attrs[key] = null
-    }
-  })
-}
-
-const toSerializedJSON = function (attrs) {
-  const { isDom } = $dom
+export const LogUtils = {
+  // mutate attrs by nulling out
+  // object properties
+  reduceMemory: (attrs) => {
+    return _.each(attrs, (value, key) => {
+      if (_.isObject(value)) {
+        attrs[key] = null
+      }
+    })
+  },
 
-  const stringify = function (value, key) {
-    if (BLACKLIST_PROPS.includes(key)) {
-      return null
-    }
+  toSerializedJSON (attrs) {
+    const { isDom } = $dom
 
-    if (_.isArray(value)) {
-      return _.map(value, stringify)
-    }
+    const stringify = function (value, key) {
+      if (BLACKLIST_PROPS.includes(key)) {
+        return null
+      }
 
-    if (isDom(value)) {
-      return $dom.stringify(value, 'short')
-    }
+      if (_.isArray(value)) {
+        return _.map(value, stringify)
+      }
 
-    if (!(!_.isFunction(value) || !groupsOrTableRe.test(key))) {
```

Comment:
I changed this part to `_.isFunction(value) && groupsOrTableRe.test(key)`, because it's a bit hard to read with double negation.